x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/utils.ts

```diff
@@ -20,6 +20,10 @@ export const getUpdatedDeploymentType = (newPolicy: NewPackagePolicy, inputType:
     inputs: newPolicy.inputs.map((item) => ({
       ...item,
       enabled: item.type === inputType,
+      streams: item.streams.map((stream) => ({
+        ...stream,
+        enabled: item.type === inputType,
+      })),
```

Comment:
in addition to enabling the input, now also enabling all of its streams too. 

similar to syntethics, except we map all streams of an input (no way to identify it yet, and there's only 1):

https://github.com/elastic/kibana/blob/8f51b55a31147462c1964e60c0bae40850b5a2bb/x-pack/plugins/synthetics/common/formatters/format_synthetics_policy.ts#L40-L41

this makes the difference when creating the input's streams field in the agent policy:

https://github.com/elastic/kibana/blob/8f51b55a31147462c1964e60c0bae40850b5a2bb/x-pack/plugins/fleet/server/services/agent_policies/package_policies_to_agent_inputs.ts#L63-L65

so now when comparing agent policies for an EKS integration saved with/without our extension, the output is the same except for various IDs and names. the streams array is populated with the same values.